app/controllers/api/v2/green_lanes/subheadings_controller.rb

```diff
@@ -4,8 +4,9 @@ module GreenLanes
       class SubheadingsController < BaseController
         def show
           subheading = ::GreenLanes::FetchSubheadingsService.new(params[:id]).call
-          presented_subheading = SubheadingPresenter.new(subheading)
-          serializer = Api::V2::GreenLanes::SubheadingSerializer.new(presented_subheading, include: %w[applicable_measures])
+          possible_categorisations = ::GreenLanes::CategorisationsService.new.find_possible_categorisations(subheading)
```

Comment:
1. should we use the verb in the name of the service (like other services?),
for example `FindCategorizationsService`?

2. Also,  should we rename `possible_categorisations` to just  `categorisations`?
is the "possible" because the result could be empty? (or there is a semantic reasoning behind it?)

Reply:
Renamed the service but leave the field name until we get a resolution